app/src/main/java/org/wikipedia/yearinreview/YearInReviewSlides.kt

```diff
@@ -81,13 +84,27 @@ class YearInReviewSlides(
         )
     }
 
-    private fun readingPatternsScreen(vararg params: Int): YearInReviewScreenData.StandardScreen {
-        // TODO: yir106 + yir107
-        return YearInReviewScreenData.StandardScreen(
-            animatedImageResource = R.drawable.year_in_review_puzzle_pieces,
-            staticImageResource = R.drawable.year_in_review_puzzle_pieces,
-            headlineText = "You have clear reading patterns",
-            bodyText = "TBD"
+    private fun readingPatternsScreen(): YearInReviewScreenData.StandardScreen {
+        // TODO: check if the time needs to follow the locale
+        val favoriteTimeText = when (yearInReviewModel.favoriteTimeToRead) {
```

Comment:
Pending. Please share other ideas on how to set up the hour based on the timezone properly.